website/app/components/doc/table-of-contents/collapsible-item.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="doc-table-of-contents__folder">
+  <button
+    type="button"
+    aria-controls={{this.contentId}}
```

Comment:
question: should we only render `aria-controls` if expanded? otherwise it would point to a non-existent element